shap/plots/_bar.py

```diff
@@ -392,6 +392,13 @@ def bar(
 
 
 def bar_legacy(shap_values, features=None, feature_names=None, max_display=None, show=True):
+    warnings.warn(
+        "The behaviour of this function will change in a future version to the new plotting API."
```

Comment:
will this change or is this a candidate for removal?

Reply:
That's a good question, I was thinking the same. I propose switching `shap.bar_plot` to point to `shap.plots.bar`

Reply:
Havent thought this through, but could be an idea to pursue